docs/development/go/go-structures/index.md

```diff
@@ -394,15 +698,29 @@ func main() {
         },
     }
 
+    fmt.Println("struct saved to JSON":)
     saveToJSON(os.Stdout, myRecord)
 }
 {{< /file >}}
 
-The `loadFromJSON()` function is used for decoding the data of a JSON file according to a data structure that is given as the second argument to it. We first call `json.NewDecoder()` to create a new JSON decode variable that is associated with a file and then we call the `Decode()` function for actually decoding the contents of the file and putting them into the desired variable. The function uses the empty interface type (`interface{}`) in order to be able to accept any data type – you will learn more about interfaces in a forthcoming guide. The `saveToJSON()` function creates a JSON encoder variable named `encodeJSON`, which is associated with a filename, which is where the data is going to be put. The call to `Encode()` is what puts the data into the desired file after encoding it. In our case `saveToJSON()` is called using `os.Stdout`, which means that data is going to standard output. Last, the `myRecord` variable contains sample data using the `Record` and `Telephone` structures defined at the beginning of the program. It is the contents of the `myRecord` variable that are processed by `saveToJSON()`.
+- The `loadFromJSON()` function is used for decoding the data of a JSON file according to a data structure that is given as the second argument to it.
```

Comment:
This is basically all the same, but I split the explanation into a bulleted list to make it a bit easier to read.